CFC_DataCollector/trip_generator/input.json

```diff
@@ -0,0 +1,15 @@
+{
+	"radius" : "1",
+	"starting centroids" :
+	[
+		"2703 Hallmark Dr Belmont Ca",
+		"ATT Park"
+
+	],
+	"ending centroids" :
+	[
+		"UC Berkeley",
+		"Morgan Hill"
+	],
+	"number of trips" : 100
```

Comment:
Maybe you could allow people to specify the probability of taking a trip to or from specific places?